### PR TITLE
[ITSASU-1309] Convert to hmrc-mongo

### PR DESCRIPTION
**Pull request by aug24**

### CODE REVIEW

**Comment by kayyala** [comment]
Hi there can you please remove from your GitHub repo I’m still getting mail

**Comment by AlexRimmerHMRC** [comment]
> Hi there can you please remove from your GitHub repo I’m still getting mail

You can click to unwatch the repository

File: test/controllers/SubscriptionDataControllerSpec.scala [hunk 1]
```diff
@@ -162,7 +163,8 @@ class SubscriptionDataControllerSpec extends CommonSpec with MockSubscriptionDat
     "return OK" when {
       "the data related to the given sessionId have been deleted successfully and an OK status returned from the service" in {
         mockAuthSuccess()
-        mockDeleteSessionData(reference)(UpdateWriteResult(ok = true, 1, 1, Seq(), Seq(), None, None, None))
+        mockDeleteSessionData(reference)(DeleteResult.acknowledged(1))
+        //ok = true, 1, 1, Seq(), Seq(), None, None, None))
```
Comment (sb-dev) [inline test/controllers/SubscriptionDataControllerSpec.scala]: The comments can be removed?

Comment (AlexRimmerHMRC) [reply test/controllers/SubscriptionDataControllerSpec.scala]: Done

**Review by sb-dev — APPROVED** [review]

**Pull request merged**